ui/components/app/network-account-balance-header/network-account-balance-header.js

```diff
@@ -45,13 +46,15 @@ export default function NetworkAccountBalanceHeader({
           alignItems={ALIGN_ITEMS.CENTER}
         >
           <Identicon address={accountAddress} diameter={32} />
-          <Identicon
-            address={accountAddress}
-            diameter={16}
-            imageBorder
-            image="./images/eth_badge.svg"
-            className="network-account-balance-header__network-account__ident-icon-ethereum"
-          />
+          {tokenName === CURRENCY_SYMBOLS.ETH ? (
+            <Identicon
```

Comment:
I think `IconWithFallback` will achieve the desired UX if there is no icon image for the network